lisp/ein-company.el

```diff
@@ -110,7 +110,7 @@
      (let* ((kernel (ein:get-kernel-or-error))
             (cached (ein:completions-get-cached arg (ein:$kernel-oinfo-cache kernel))))
        (ein:aif cached it
-         (unless (ein:company--punctuation-check (thing-at-point 'line) (current-column))
+         (unless (and (looking-at "[[:nonascii:]]") (ein:company--punctuation-check (thing-at-point 'line) (current-column)))
```

Comment:
Let looking-at-nonascii be A, and punctuation-check be B.
I think the change effectively makes the line a no-op since if A is true then B will be false, making "if (~A | ~B)" always true.  Did you test the change?